src/providers/openstack/network.rs

```diff
@@ -22,7 +31,21 @@ impl OpenstackProviderNetwork {
     }
 
     fn endpoint_for(key: &str) -> String {
```

Comment:
Since we now have two classes of endpoints, it'd be clearer to rename this to e.g. `ec2_endpoint_for`.